db/migrate/20180726220853_create_amr_counts.rb

```diff
@@ -0,0 +1,16 @@
+class CreateAmrCounts < ActiveRecord::Migration[5.1]
+  def change
+    create_table :amr_counts do |t|
+      t.string :gene
+      t.float :coverage
+      t.float :depth
+      t.bigint :pipeline_run_id
+      t.string :drug_family
+      t.integer :level
+      t.float :drug_gene_coverage
+      t.float :drug_gene_depth
+      t.timestamps
+    end
+    add_index :amr_counts, :allele, unique: true
```

Comment:
This isn't the right unique index to add. This would mean there could only be 1 entry for each allele in the entire table. Instead, it should be restricted to one entry per allele PER PIPELINE RUN. So: `add_index :amr_counts, [:pipeline_run_id, :allele], unique: true`

Reply:
thanks!